assignments/callbacks.js

```diff
@@ -21,22 +21,42 @@ const items = ['Pencil', 'Notebook', 'yo-yo', 'Gum'];
 
 */
 
-
-function getLength(arr, cb) {
-  // getLength passes the length of the array into the callback.
+// getLength passes the length of the array into the callback.
+ 
+function getLength(arr, cb) {  return cb(arr.length);
 }
+getLength(items, function(length) {
+ console.log(length);
+});
+
 
 function last(arr, cb) {
-  // last passes the last item of the array into the callback.
+  return cb(arr[arr.length-1]);
 }
+  last(items, log);
```

Comment:
Unfortunately, this doesn't return for you bc you don't have the same log function that Matt used. 
You can make this work by doing exactly what you did above and making it call items and a function.

last(items, function(lastItem) {
//   console.log(lastItem);
// });